packages/chrome/src/utils/fetch.ts

```diff
@@ -0,0 +1,19 @@
+export function get<T>(url: RequestInfo, options?: RequestInit) {
+  return fetch(url, { ...options, method: 'GET' })
+    .then((res) => res.json())
+    .then(({ data }: { data: T }) => data);
+}
+
+export function post<R, T extends Record<string, any> = Record<string, any>>(url: RequestInfo, data: T, options?: Omit<RequestInit, 'body'>) {
+  return fetch(url, {
+    ...options,
+    headers: {
+      ...options?.headers,
+      'Content-Type': 'application/json',
+    },
+    method: 'POST',
```

Comment:
You should also add `credentials: 'include'` to include creds https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#sending_a_request_with_credentials_included